libevm/VM.cpp

```diff
@@ -25,6 +25,19 @@ using namespace std;
 using namespace dev;
 using namespace dev::eth;
 
+
+// real machine word, virtual machine word, signed and unsigned overflow words
```

Comment:
rmword is whatever is best performance for the real machine.  Typically uint64_t.  For isolating some bugs it can be helpful to set it to u256 or u512.